apps/02RockPaperScissors.js

```diff
@@ -4,11 +4,40 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
-
 function rockPaperScissors(hand1, hand2) {
-
-    // Write code here
-
+  if (hand1 === hand2) {
```

Comment:
Well done Kayla! I will show how to shrink the code down in the next class a bit but it's important that you're comfortable with covering all cases using conditionals.